simulation/simulation_linthompsamp.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.lines as mlines
+import sys
+sys.path.append("..")
+from striatum.storage import history
+from striatum.storage import model
+from striatum.bandit import linthompsamp
+
+
+class LinThompSampLinearPayoff:
+    def __init__(self, t, d, actions):
+        self.T = t
+        self.d = d
+        self.actions = actions
+
+    def data_simulation(self):
```

Comment:
There're several methods look almost the same among these simulation classes. I suggest to pull them out as single functions or build an abstract class.